core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -55,32 +63,41 @@ public static RocksPageStore open(RocksPageStoreOptions pageStoreOptions) {
     Preconditions.checkArgument(pageStoreOptions.getMaxPageSize() > 0);
     RocksDB.loadLibrary();
     // The RocksObject will be closed together with the RocksPageStore
-    Options rocksOptions = new Options()
+    DBOptions rocksOptions = new DBOptions()
```

Comment:
also check if this is closed

Reply:
From the javadoc on `RocksDB.open`:

> Options instance *should* not be disposed before all DBs using this options instance have been closed. If user doesn't call options dispose explicitly, then this options instance will be GC'd automatically

Make sure to keep the reference to the options object live, and close it until the DB is closed.

Reply:
Yep, as @dbw9580 suggested, I just close the DBOptions on the db close.  Thank you!